pom.xml

```diff
@@ -327,6 +328,13 @@
         <scope>test</scope>
       </dependency>
 
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
Is `junit-jupiter-engine` absolutely necessary?

Reply:
It is JUnit 5 that is used in tests. And it is not going in the final jar because of scope `test`